source/chapters/user_interface.rst

```diff
@@ -666,15 +676,28 @@ Equalizers and Gain Knobs
    EQ Controls
 
 **Gain Knob**
-  Above these knobs, the gain knob allows you to adjust the gain of the deck. In
-  general, you should adjust this knob so that at full-volume the deck's audio
-  is just at the peak of the center VU meters. This is so you can achieve the
-  widest dynamic range for your track.
+  Above these knobs, the gain knob allows you to adjust the gain of the deck. 
+  Use this to compensate for the differences in recording levels between tracks.
+  In general, you should adjust this knob so that at full-volume the deck's
+  audio is just at the peak of the center VU meters. This is so you can achieve
```

Comment:
The same as above, do not use the yellow region. This is the headroom for effects, mixing and dynamic peaks of the tracks.